kairon/shared/data/action_serializer.py

```
@@ -0,0 +1,467 @@
+    @staticmethod
+    def validate(bot: str, actions: dict, other_collections: dict):
+        """
+        Validates the action configuration data, first return parameter is true if validation is successful
+        :param bot: bot id
+        :param actions: action configuration data
+        :param other_collections: other collection configuration data
+        :return: is_data_valid: bool, error_summary: dict, component_count: dict
+        """
+        is_data_invalid = False
+        component_count = dict.fromkeys(ActionSerializer.action_lookup.keys(), 0)
+        error_summary = {key: [] for key in ActionSerializer.action_lookup.keys()}
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)